AutomatedTesting/Gem/PythonTests/editor/EditorScripts/Docking_BasicDockedTools.py

```diff
@@ -50,14 +55,19 @@ async def run_test(self):
         :return: None
         """
 
-        # Create a level since we are going to be dealing with an Entity.
-        self.create_level(
-            self.args["level"],
-            heightmap_resolution=1024,
-            heightmap_meters_per_pixel=1,
-            terrain_texture_resolution=4096,
-            use_terrain=False,
-        )
+        from PySide2 import QtWidgets, QtTest, QtCore
+
+        import azlmbr.legacy.general as general
+        import azlmbr.bus as bus
+        import azlmbr.editor as editor
+        import azlmbr.entity as entity
+
+        from editor_python_test_tools.utils import Report
+        from editor_python_test_tools.utils import TestHelper as helper
+
+        # Open an existing simple level
+        helper.init_idle()
+        helper.open_level("Physics", "Base")
```

Comment:
May be worth for the future to create a "Base" or "Empty" level in the root level folder.
Doesn't make a lot of sense to be in the Physics folder

Reply:
Yeah, but makes more sense for us to eventually move this to the root path and not in Physics/